core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -65,8 +65,8 @@ private[spark] trait TaskScheduler {
    * alive. Return true if the driver knows about the given block manager. Otherwise, return false,
    * indicating that the block manager should re-register.
    */
-  def executorHeartbeatReceived(execId: String, taskMetrics: Array[(Long, TaskMetrics)],
-    blockManagerId: BlockManagerId): Boolean
+  def executorHeartbeatReceived(execId: String, executorMetrics: ExecutorMetrics,
+      taskMetrics: Array[(Long, TaskMetrics)], blockManagerId: BlockManagerId): Boolean
```

Comment:
I know this was wrong before, but as long as you're touching this -- with multiline methods, each arg should be on its own line.